include/dynd/types/tuple_type.hpp

```diff
@@ -145,13 +147,13 @@ namespace ndt {
     }
 
     /** Makes a tuple type with the specified types */
-    static type make(const nd::array &field_types, bool variadic = false)
+    static type make(std::vector<type> field_types, bool variadic = false)
     {
       return type(new tuple_type(field_types, variadic), false);
```

Comment:
Yes, I'm okay with that. I do think we should do const std::vector & now, because that makes a lot of things easier (with regard to passing temporaries and the like).